net-p2p/qbittorrent/metadata.xml

```diff
@@ -1,17 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<maintainer type="person">
+	<maintainer type="person" proxied="proxy">
 		<email>eschwartz@gentoo.org</email>
 		<name>Eli Schwartz</name>
 	</maintainer>
+	<maintainer type="person" proxied="yes">
+		<email>fkobi@pm.me</email>
+		<name>Filip Kobierski</name>
+	</maintainer>
 	<use>
 		<flag name="webui">
 			Install qBittorrent Web UI (qbittorrent-nox) and systemd/OpenRC service
 		</flag>
 	</use>
 	<upstream>
-		<remote-id type="github">qbittorrent/qBittorrent</remote-id>
+		<bugs-to>https://github.com/qbittorrent/qBittorrent/issues</bugs-to>
 		<changelog>https://www.qbittorrent.org/news</changelog>
+		<doc>https://github.com/qbittorrent/qBittorrent/wiki</doc>
+		<remote-id type="github">qbittorrent/qBittorrent</remote-id>
+		<remote-id type="sourceforge">qbittorrent</remote-id> <!-- Just for distfiles -->
```

Comment:
Is there a comment that pkgcheck will use to suppress the warning? If not, maybe that would be a useful pkgcheck feature request?

Otherwise I don't really care about a comment. I guess you could add one if you like.